buildlib/az-distro-release.yml

```diff
@@ -71,11 +70,20 @@ jobs:
           echo 10 > debian/compat   # https://www.debian.org/doc/manuals/maint-guide/dother.en.htmdpl#compat
           dpkg-buildpackage -us -uc -Pcuda
           cd ..                             # Move back to the working directory
-          find . -name '*.deb'
-          VER="${POSTFIX#ucx-}"   # Remove 'ucx' prefix from the POSTFIX string
-          # Rename DEB files
+
+          # Rename DEB files          
+          VER="${POSTFIX#ucx-}"     # Remove 'ucx-' prefix from the POSTFIX string
           find . -name "ucx*.deb" -exec bash -c 'mv "$1" "${1%%_*}-'"${VER}"'.deb"' _ {} \;
+          find . -name '*.deb'      # Show new names
+
+          # Remove superfluous dependency
+          dpkg-deb -R "ucx-cuda-${VER}.deb" tmp    # Extract
+          sed -i 's/libnvidia-compute-[0-9]* | libnvidia-ml1, //g' tmp/DEBIAN/control
+          dpkg-deb -b tmp "ucx-cuda-${VER}.deb"        # Rebuild
+          dpkg-deb -I "ucx-cuda-${VER}.deb"
```

Comment:
Future improvement: strict check for deps - make sure there're only expected deps

Reply:
It's a good idea to include such a test. Two deps expected in this specific case.

Reply:
Please put a task for this into our backlog